src/main/java/com/google/jenkins/plugins/credentials/oauth/JsonServiceAccountConfig.java

```diff
@@ -53,36 +63,97 @@
   private static final long serialVersionUID = 6818111194672325387L;
   private static final Logger LOGGER =
       Logger.getLogger(JsonServiceAccountConfig.class.getSimpleName());
-  private String jsonKeyFile;
+  @CheckForNull
+  private String filename;
+  @CheckForNull
+  private SecretBytes secretJsonKey;
+  @Deprecated   // for migration purpose
+  @CheckForNull
+  private transient String prevJsonKeyFile;
```

Comment:
The field name should be preserved to migrate from old configurations.

Reply:
Thanks for pointing that out. I'll change the name for the file item in the setters and getters and jelly config instead and leave this one alone